mkdocs.yml.tpl

```diff
@@ -7,6 +7,7 @@ repo_url: https://github.com/cozy/
 edit_uri: cozy.github.io/edit/dev/src
 site_description: Cozy documentation
 site_author: CozyCloud - https://cozy.io
+copyright: '<a href="/en/privacy">privacy</a>'
```

Comment:
```suggestion
privacy: '<a href="/en/privacy">privacy</a>'
```

Reply:
nope.

copyright is a supported feature of our theme to put a notice in footer of each page.

Privacy will be ignored and is useless keyword